src/traffic_server/InkAPI.cc

```diff
@@ -9306,6 +9306,26 @@ TSVConnSslConnectionGet(TSVConn sslp)
   return ssl;
 }
 
+const char *
+TSSslSNIGet(TSVConn sslp, int *length)
+{
+  char const *server_name   = nullptr;
+  NetVConnection *vc        = reinterpret_cast<NetVConnection *>(sslp);
+  SSLNetVConnection *ssl_vc = dynamic_cast<SSLNetVConnection *>(vc);
```

Comment:
We don't need SSLNetVC here because `get_server_name()` is declared in `NetVConnection`.